internal/i18n/en_translations_descriptions.go

```diff
@@ -0,0 +1,544 @@
+	TokenApprovalCreated         = ffm("TokenApproval.created", "The creation time of the token approval")
+	TokenApprovalTX              = ffm("TokenApproval.tx", "If submitted via FireFly through the local node, this will reference the UUID of the FireFly transaction")
+	TokenApprovalBlockchainEvent = ffm("TokenApproval.blockchainEvent", "The UUID of the blockchain event")
+	TokenApprovalConfig          = ffm("TokenApproval.config", "Input only field, with token connector specific configuration of the approval.  See your chosen token connector documentation for details")
+
+	// TokenApprovalInputInput field descriptions
+	TokenApprovalInputInputPool = ffm("TokenApprovalInput.pool", "The name or UUID of a token pool")
+
+	// TokenBalance field descriptions
+	TokenBalancePool       = ffm("TokenBalance.pool", "The UUID the token pool this balance entry applies to")
+	TokenBalanceTokenIndex = ffm("TokenBalance.tokenIndex", "The index of the token within the pool that this balance applies to")
+	TokenBalanceURI        = ffm("TokenBalance.uri", "The URI of the token this balance entry applies to")
+	TokenBalanceConnector  = ffm("TokenBalance.connector", "The token connector that is responsible for the token pool of this balance entry")
+	TokenBalanceNamespace  = ffm("TokenBalance.namespace", "The namespace of the token pool for this balance entry")
+	TokenBalanceKey        = ffm("TokenBalance.key", "The blockchain signing identity this balance applies to")
+	TokenBalanceBalance    = ffm("TokenBalance.balance", "The numeric balance. For non-fungible tokens will always be 1. For fungible tokens, the number of decimals for the token pool should be considered when interpreting the balance. For example, with 18 decimals a fractional balance of 10.234 will be returned as 10,234,000,000,000,000,000")
+	TokenBalanceUpdated    = ffm("TokenBalance.updated", "The last time the balance was updated by applying a transfer event")
+
+	// TokenBalance field descriptions
+	TokenConnectorName = ffm("TokenConnector.name", "The name of the token connector, as configured in the FireFly core configuration file")
+
+	// TokenPool field descriptions
+	TokenPoolID         = ffm("TokenPool.id", "The UUID of the token pool")
+	TokenPoolType       = ffm("TokenPool.type", "The type of token the pool contains, such as fungible/non-fungible")
+	TokenPoolNamespace  = ffm("TokenPool.namespace", "The namespace for the token pool")
+	TokenPoolName       = ffm("TokenPool.name", "The name of the token pool. Note the name is not validated against the description of the token on the blockchain")
+	TokenPoolStandard   = ffm("TokenPool.standard", "The ERC standard the token pool conforms to, as reported by the token connector")
+	TokenPoolProtocolID = ffm("TokenPool.protocolId", "The protocol identifier of the blockchain event that established the existence of the pool on-chain. See the 'protocolId' field on blockchain events for more information")
```

Comment:
This is actually _not_ the same as the blockchain protocol ID. Format is totally up to the connector. Suggestion: `An identifier for the pool, as provided by the token connector.`

For the record, erc1155 uses something like "F1" and erc20 uses something like "address=0x123&schema=erc20" (not sure if examples are worth calling out here though).